backend-shared/config/db/application_test.go

```diff
@@ -51,6 +55,7 @@ var _ = Describe("Application Test", func() {
 			Engine_instance_inst_id: gitopsEngineInstance.Gitopsengineinstance_id,
 			Managed_environment_id:  managedEnvironment.Managedenvironment_id,
 			SeqID:                   int64(seq),
+			Created_on:              applicationget.Created_on,
```

Comment:
It would be good to instead assign Created_on a new value, and make sure that update correctly updates this field to a new value.

Presuming you agree, we should make the same change to the other tests as well.

Reply:
@jgwest I assumed that we don't have to add  new value to Created_on field for update as Created_on field contains a time  when the Application was `created` for eg, if Application was created at 9:05 PM then Created_on field should be 9:05 PM , and if we are updating the same Application at 9:15 PM then we can keep the Created_on field as it is rather than addding a new value(9:15 PM) because Application was created at 9:05 PM(as Created_on field specifies that when the Application was created ), I think we can add new field called Updated_on so that we can see when this Application was updated, please correct me if I am assuming this wrong

Reply:
@Rizwana777 yup, that's fair, thanks for the explanation! We can leave it is as is.